src/domain/Chat/Chat.module.scss

```
@@ -149,56 +182,38 @@
     &__field {
-      padding: 12px 16px;
       flex: 1;
-      height: 48px;
+      min-height: 44px;
+      height: auto;
+      max-height: none;
+      padding: 8px 16px;
       border-radius: 24px;
-      border: 2px solid rgba(f.color(orange-600), 0.2);
+      border: 1px solid f.color(orange-500);
       background: rgba(255, 255, 255, 0.9);
```

Stylelint (17.2.0)

[error] Expected "0.9" to be "90%" (alpha-value-notation)

(alpha-value-notation)